docs/recipes/ImplementingUndoHistory.md

```diff
@@ -0,0 +1,181 @@
+# Implementing Undo History
+
+To add undo functionality (or any other functionality) to your existing
+reducers, you need to create a higher-order reducer, which is a function
```

Comment:
I think this is jumping into the topic too fast. Before the reader understands why they need a reducer => reducer, they need to understand:

* we can express undo history with a regular Redux state, but of a specific form
* we can express undo/redo as regular actions
* we can call write `undoableTodos` reducer that calls `todos` manually
* finally, let's abstract `undoable` out of it!

I understand if this is bigger than you currently have time for—I can revisit this later if you don't find the time.